### PR TITLE
fix: get auction info for tokens in one call

### PR DESCRIPTION
**Pull request by folucode**
merged the auctionInfo table with the tokenInfo table in the list-tokens endpoint to get all the auction information about a token

### CODE REVIEW

File: yasuke-server/src/services/token.service.ts [hunk 1]
```diff
@@ -68,20 +78,37 @@ export class TokenService {
   }
 
   async listTokens(
```
Comment (segun) [inline yasuke-server/src/services/token.service.ts:80]: Can we leave listTokens method as is and create a new method, `listTokensWithAuction` (or something like that) for this change.

That means the controller will also have to expose another endpoint for this purpose. and the GUI guys will need to change the call to listTokensWithAuction whenever they need tokenInfo with auction info.

**Comment by segun** [comment]
We need to standardise formatting as most of the changes in this PR are formatting changes. Can you create a .prettierrc (I assume you are using prettier) and push to the repo. Thanks...Good work!

File: yasuke-server/src/services/token.service.ts (hunk 1 above)
Comment (folucode) [reply yasuke-server/src/services/token.service.ts:80]: Okay. I'll do this. thank you.

**Comment by folucode** [comment]
@segun I've made the changes you requested to the endpoint. Pleas check it. Thanks

**Review by segun — APPROVED** [review]

**Pull request merged**